src/analysis/code_analysis.md

```diff
@@ -289,29 +316,30 @@ sub.strlen_d50 0x5de0 [STRING] lea rcx, str.02x
 (nofunc) 0x17838 [CODE] jae str.02x
 ```
 
-There are also some useful commands under `axt`. Use `axtg` to generate rizin commands which will help you to create graphs according to the XREFs.
+There are also some useful commands under `axt`. Use `axtg` to generate rizin commands which will help you to create
+graphs according to the XREFs.
 
 ```
 [0x08048320]> s main
 [0x080483e0]> axtg
-agn 0x8048337 "entry0 + 23"
-agn 0x80483e0 "main"
-age 0x8048337 0x80483e0
+agn 0x1118 "entry0 + 24"
+agn 0x11e9 "main"
+age 0x1118 0x11e9
```

Comment:
these changes are a bit weird, because the address shown is `0x08048320` and `0x080483e0` but `axtg` shows `0x1118`